fms2_io/fms2_io.F90

```diff
@@ -214,17 +214,20 @@ module fms2_io_mod
 
 logical, private :: fms2_io_is_initialized = .false. !< True after fms2_io_init is run
 !< Namelist variables
-integer :: ncchksz = 64*1024  !< User defined chunksize (in bytes) argument in netcdf file 
+integer :: ncchksz = 64*1024  !< User defined chunksize (in bytes) argument in netcdf file
                               !! creation calls. Replaces setting the NC_CHKSZ environment variable.
+character (len = 10) :: netcdf_default_format = "64bit" !< User defined netcdf file format, acceptable values
+                              !! are: "64bit", "classic", "netcdf4". This can be overwritten if you specify
+                              !! "nc_format" in the open_file call
 namelist / fms2_io_nml / &
-                      ncchksz
+                      ncchksz, netcdf_default_format
```

Comment:
This is likely to conflict with #427

Reply:
definitely! I can resolve the conflicts after either PR is merged...